cmd/kube-controller-manager/controller-manager.go

```diff
@@ -109,8 +117,8 @@ func main() {
 			api.ResourceMemory: *nodeMemory,
 		},
 	}
-	nodeController := nodeControllerPkg.NewNodeController(cloud, *minionRegexp, machineList, nodeResources, kubeClient)
-	nodeController.Run(*nodeSyncPeriod)
+	nodeController := nodeControllerPkg.NewNodeController(cloud, *minionRegexp, machineList, nodeResources, kubeClient, kubeletClient)
+	nodeController.Run(*nodeSyncPeriod, 10)
```

Comment:
Hmm...well, we are using magic number quite a frew places, so I haven't pay much attention to it.  Changed to a flag.